### ISSUE TITLE
Error with undefined properties (regression from v1.2.1)

### ISSUE THREAD

**Issue by giuseppelt** [rafeca/prettyjson#58]
The commit 25a00a18c5cdbc589a8a7317bc0a068b710aa333 introduced a regression from previous behavior.
```
prettyjson.render({
  prop1: "value",
  prop2: undefined
});
```
throws an unhandled error (RangeError: Invalid array length), while the previous version did not, it just skipped the property instead.

Actually this blocks all https://github.com/netlify/cli deploy commands, as it depends on this package.

**Comment by rafeca** [comment]
Hey! I can't reproduce the issue: the snipped that you passed works fine (it prints now the undefined value). I'm wondering if the error comes from the netlify CLI that cannot handle the undefined somehow?

Anyways, I'm going to add an option param to control this, and make undefined values hidden by default since I think it's a better behaviour

**Comment by rafeca** [comment]
I've published v1.2.5 which ignores undefined values unless `renderUndefined` is passed.

**Comment by giuseppelt** [comment]
It works now. Thanks for the fix.
